index/scorch/segment/zap/contentcoder.go

```diff
@@ -94,9 +93,27 @@ func (c *chunkedContentCoder) flushContents() error {
 		return err
 	}
 
+	// convert the document data lens to data offsets.
+	// This enables the direct look up for the start,end
+	// offsets for a given document entry inside a chunk.
+	// Starting offset of i'th index is stored in i-1'th
+	// position except for 0'th index and ending offset is
+	// stored at i'th index position.
```

Comment:
I find this sentence confusing.  It seems to say that the offset of the 1st index is in position 0 (i-1) but that the offset of the 0th index is also in position 0 (i).

Reply:
done!